forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -928,25 +940,19 @@ export const RecordsLog = ({
       bcgovcode: bcgovcode,
       attributes: [],
     };
-
     //remove duplicate records(except duplicate attachments)
     const deduplicatedRecords = removeDuplicateFiles(recordList);
     //get only relevent fields
-    const updatedrecords = getUpdatedRecords(
-      deduplicatedRecords,
-      conversionFormats
-    );
+    const updatedrecords = getUpdatedRecords(deduplicatedRecords);
```

Comment:
removed conversionFormats parameter as it is not needed. Part of pdfstitch attachment fix